### PR TITLE
Change long to long long & clean daal versions in examples

### PR DESCRIPTION
**Pull request by Pahandrovich**

### CODE REVIEW

File: examples/decision_forest_classification_batch.py [hunk 1]
```diff
@@ -55,7 +54,7 @@ def main(readcsv=read_csv, method='defaultDense'):
     # Traiing result provides (depending on parameters) model, outOfBagError, outOfBagErrorPerObservation and/or variableImportance
 
     # Now let's do some prediction
-    if daal_version < (2020,1):
+    if get_daal_version() < (2020,'P',1):
```
Comment (PetrovKP) [inline examples/decision_forest_classification_batch.py:57]: I prefer to delete it

File: examples/decision_forest_classification_batch.py [hunk 2]
```diff
@@ -77,7 +76,7 @@ def main(readcsv=read_csv, method='defaultDense'):
     print("\nVariable importance results:\n", train_result.variableImportance)
     print("\nOOB error:\n", train_result.outOfBagError)
     print("\nDecision forest prediction results (first 10 rows):\n", predict_result.prediction[0:10])
-    if daal_version >= (2020,1):
+    if get_daal_version() >= (2020,'P',1):
```
Comment (PetrovKP) [inline examples/decision_forest_classification_batch.py:79]: I prefer to delete it

File: examples/gradient_boosted_classification_batch.py [hunk 3]
```diff
@@ -41,9 +43,7 @@ def main(readcsv=read_csv, method='defaultDense'):
 
     # Configure a training object (5 classes)
     # previous version has different interface
-    from daal4py import _get__daal_link_version__ as dv
-    daal_version = tuple(map(int, (dv()[0:4], dv()[4:8])))
-    if daal_version < (2020,0):
+    if get_daal_version() < (2020,'P',0):
```
Comment (PetrovKP) [inline examples/gradient_boosted_classification_batch.py:46]: I prefer to delete it

File: src/oneapi/oneapi.pyx [hunk 4]
```diff
@@ -126,6 +126,6 @@ cdef api object make_py_from_sycltable(void * ptr, int typ, int d1, int d2):
     cdef void * buff = c_make_py_from_sycltable(ptr, typ)
     if buff:
         res = sycl_buffer.__new__(sycl_buffer)
-        res.__inilz__(<long>buff, typ, d1, d2)
+        res.__inilz__(<long long>buff, typ, d1, d2)
```
Comment (michael-smirnov) [inline src/oneapi/oneapi.pyx:129]: Thanks!

**Review by michael-smirnov — APPROVED** [review]

**Review by PetrovKP — APPROVED** [review]

**Pull request merged**